rewrite-maven/src/main/java/org/openrewrite/maven/MavenSettings.java

```diff
@@ -65,6 +77,7 @@ public class MavenSettings {
     Servers servers;
 
     @JsonCreator
+    @ParametersAreNullableByDefault
```

Comment:
This has the split package problem that JSR305 has in general and cannot be used in a modular context (yay Jigsaw).